donghyun/lectureProblem/binarySearch/binarySearchBascics.js

```diff
@@ -0,0 +1,23 @@
+const makeFloorMid = (start, end) => Math.floor((start + end) /2);
+
+const binarySearch = (input) => {
+    const inputData = input.split("\n");
+    const [length, findIndex] = inputData.shift().split(" ").map(Number);
+    const ASCNumbersList = inputData[0].split(" ").sort();
```

Comment:
```suggestion
    const [line1, line2] = input.split("\n");
    const [length, findIndex] = line1.split(" ").map(Number);
    const ASCNumbersList = line2.split(" ").sort();
```
여기 왜 0 index 보나했는데 shift있었네 ㅋㅋㅋ 
이럴떈 그냥 위에서 분리하고 사용하는게 덜 헷갈릴듯

Reply:
"8 32\n23 87 65 12 57 32 99 81" 여기서 \n 짜르고 shift() 하면 뒤에 ["23 87 65 12 57 32 99 81"] 남아서 그냥 split(" ").sort(); 하면 될줄알았는데 이미 배열이라 index 지정해줘야하더라 어케분리해야하는지 모르겠어서 걍 했는데  위에서 아예 바꿔ㅏ봐야겠농